examples/widgets/flutter.yaml

```diff
@@ -1,4 +1,11 @@
 name: widgets
+assets:
+  - assets/world.meta
+  - assets/1.0x/world.png
+  - assets/1.5x/world.png
+  - assets/2.0x/world.png
+  - assets/3.0x/world.png
+  - assets/4.0x/world.png
```

Comment:
I will note that adding more logic to the tooling implies more opinionated asset management at lower levels. I wanted to be somewhat careful about forcing authors to use an asset layout strategy that might not fit their needs, particularly as the qualifier system here only handles a relatively narrow use case.